Source/Core/Core/IOS/IOSC.h

```diff
@@ -17,6 +17,84 @@ class PointerWrap;
 
 namespace IOS
 {
+enum class SignatureType : u32
+{
+  RSA4096 = 0x00010000,
+  RSA2048 = 0x00010001,
+  // XXX: Add support for ECC (0x00010002).
+};
+
+enum class PublicKeyType : u32
+{
+  RSA4096 = 0,
+  RSA2048 = 1,
+};
+
+#pragma pack(push, 4)
+struct SignatureRSA4096
+{
+  SignatureType type;
+  u8 sig[0x200];
+  u8 fill[60];
+  char issuer[0x40];
```

Comment:
You're right, it would be better not to mix hex notation with decimal. (All of these structs were reorganised a bit and split/merged as I was looking into IOSC and cross checking with libogc.) Fixed.